packages/rum-core/src/domain/view/trackViews.spec.ts

```diff
@@ -897,4 +906,49 @@ describe('view event count', () => {
       view: viewTest.getLatestViewContext(),
     } as RumEvent & Context
   }
+
+  describe('update view name', () => {
+    let setupBuilder: TestSetupBuilder
+    let viewTest: ViewTest
+
+    beforeEach(() => {
+      setupBuilder = setup().beforeBuild((buildContext) => {
+        viewTest = setupViewTest(buildContext)
+        return viewTest
+      })
+    })
+
+    afterEach(() => {
+      resetExperimentalFeatures()
+    })
+
+    it('should update an undefined view name if the experimental feature is enabled', () => {
+      addExperimentalFeatures([ExperimentalFeature.UPDATE_VIEW_NAME])
```

Comment:
**💬 suggestion:** ‏use `mockExperimentalFeatures`, so you don't have to call `resetExperimentalFeatures`